model_compression_toolkit/core/pytorch/back2framework/pytorch_model_builder.py

```diff
@@ -188,14 +210,27 @@ def _add_modules(self):
         for n in self.node_sort:
             if isinstance(n, FunctionalNode):
                 # for functional layers
-                setattr(self, n.name, self.wrapper(n, n.type))
+                setattr(self, n.name,
+                        self.wrapper(n,
+                                     n.type,
+                                     include_activation_quantizers=not self.use_activation_holder_during_model_building)
+                        )
             else:
                 if n.type == BufferHolder:
                     self.add_module(n.name, node_builder(n))
                     self.get_submodule(n.name). \
                         register_buffer(n.name, torch.Tensor(n.get_weights_by_keys(BUFFER)).to(get_working_device()))
                 else:
-                    self.add_module(n.name, self.wrapper(n, node_builder(n)))
+                    self.add_module(n.name,
+                                    self.wrapper(n,
+                                                 node_builder(n),
+                                                 include_activation_quantizers=not self.use_activation_holder_during_model_building)
+                                    )
+            if self.get_activation_quantizer_holder is not None:
```

Comment:
use 'use_activation_holder_during_model_building'

Reply:
but activation holder can still be None

Reply:
Not sure I understand,
the implementation of use_activation_holder_during_model_building is similar to the line above